scripts/azureml-assets/setup.py

```diff
@@ -7,14 +7,15 @@
 
 setup(
    name="azureml-assets",
-   version="1.0.1",
+   version="1.1.0",

```

Comment:
Let's keep this at 1.0.1 for now so that I can get one more change into 1.1.0.
```suggestion
   version="1.0.1",

```